src/chrome/content/rules/mozillazine-fr.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="mozillazine-fr (partial)">
+
+	<target host="mozillazine-fr.org"/>
+	
+	<exclusion pattern="^http://(www\.)?cname.mozillazine-fr\.org" />
+	<test url="http://cname.mozillazine-fr.org"/>
```

Comment:
This is not necessary as long as the url is not a target host. But please add comments for non-working hosts.